ansible/config_sonic_basedon_testbed.yml

```diff
@@ -65,9 +65,15 @@
   - topo_facts: topo={{ topo }}
     delegate_to: localhost
 
+  - name: set default dualtor facts
+    set_fact:
+      dual_tor_facts: {}
+    when: "'dualtor' not in topo"
```

Comment:
Should it be like below?
```
when: "{{'dualtor' not in topo }}"
```

Reply:
I think it OK to omit the ```{{ }}``` because it's not a template. And i verified it by deploying a minigraph.

Reply:
Please ignore this. I tested your code, it works. On the contrary, ansible complains with warning `[WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: {{ 'dualtor' in topo }}` if format `when: "{{'dualtor' not in topo }}"` is used.

Reply:
Thanks @wangxin